Autopilot/Inc/gps.hpp

```diff
@@ -1,22 +1,15 @@
 /**
  * Gps sensor interface and part number selector.
- * Author: Anthony Berbari
+ * Author(s): Anthony Berbari, Dhruv Rawat
  */
 
 #ifndef GPS_HPP
 #define	GPS_HPP
 
 #include <stdint.h>
+#include "stm32f7xx_hal.h"
```

Comment:
headers should never include hal stuff. If they do, anything that uses them becomes impossible to unit test. You should include this in your cpp file instead.